core-plugins/src/main/java/io/cdap/plugin/batch/source/FTPBatchSource.java

```diff
@@ -45,9 +45,11 @@
 
 /**
  * {@link BatchSource} that reads from an FTP or SFTP server.
+ * This plugin is deprecated. Use https://github.com/data-integrations/ftp-plugins instead.
  */
+@Deprecated
 @Plugin(type = "batchsource")
-@Name("FTP")
+@Name("FTPDeprecated")
 @Description("Batch source for an FTP or SFTP source. Prefix of the path ('ftp://...' or 'sftp://...') determines " +
```

Comment:
We should change description to deprecated, use hub plugin instead.

Reply:
Thanks! I'll try that.

Reply:
fixed